shs/update_auth.sh

```
@@ -0,0 +1,23 @@
+# cd /data/project/mdwiki/
+
+rm -rf authx
+
+# Download the wd-core repository from GitHub.
+git clone https://github.com/Mdwiki-TD/auth-repo.git authx
+
+# delete composer.json and composer.lock
+rm -rf authx/composer.json authx/composer.lock
+
+# delete all json files in all subdirectories
+find authx -name *.json -delete
```

Shellcheck (0.10.0)

[warning] Quote the parameter to -name so the shell won't interpret it.

(SC2061)